docs/src/MicrophysicsFlexible.md

```diff
@@ -0,0 +1,39 @@
+
+## Moments and Sub-Distributions
+
+The prognostic variables of this parameterization are a set of N moments, which can be further divided into P sets of moments, each of which correponds to a subdistribution p. By design these moments begin at order 0 and increase as integers up to the maximum number of parameters for the chosen subdistribution. The first three such default moments have interpretable meanings:
+  - ``M_0`` - the number density of droplets [1/m^3]
+  - ``M_1`` - the mass density of droplets [kg/m^3]
+  - ``M_2`` - proportional to the radar reflectivity [kg^2/m^3]
+and can be converted to more canonical definitions of `q_liq` and `q_rai` through numerical integration.
+
+When the user wishes to use more than 2 or 3 total variables to represent the system, these moments must be divided between ``P > 1`` sub-distributions, each of which assumes the form of a particular mathematical distribution, such as an Exponential, Lognormal, or Monodisperse (each of which has two parameters), or a Gamma distribution (which takes 3 parameters). 
+
+## Loading the extension
+The package `Cloudy.jl` and its dependencies are not loaded by default when using `CloudMicrophysics.jl`. Rather, one must specify:
+```
+using CloudMicrophysics
+using Cloudy
+```
+from the Julia REPL. Upon recognizing that `Cloudy.jl` is being loaded, the extension `CloudyExt.jl` will then be loaded and overwrite the function stubs defined in `src/MicrophysicsFlexible.jl`.
+
+## Setting up a system
+All the details from the number of moments and type of subdistributions, to the parameterizations of coalescence, condensation, and sedimentation are defined through the `CLSetup` (CLoudySetup) mutable struct. This struct is mutable specifically because certain of its components, such as backend-computed coalescence tendencies, are updated prior to being passed to the timestepper. The components of a `CLSetup` object and their defaults are further described below.
+
+|   component         |   description                            |   default                  |
+|---------------------|------------------------------------------|----------------------------|
+| ``pdists``          | Vector of subdistributions corresponding | ``[Exponential, Gamma]``   |
+|                     | to the moments                           |                            |
+| ``mom``             | Prognostic moments, in the same order as | ``[0, 0, 0, 0, 0]``        |
+|                     | the corresponding subdistributions       |                            |
+| ``KernelFunc``      | Form of the coalescence kernel function  | ``LongKernelFunction``     |
+| ``mass_thresholds`` | Particle size thresholds for coalescence | ``[10.0, Inf]``            |
+|                     | integration                              |                            |
+| ``kernel order``    | Polynomial order for the approx. kernel  | ``1``                      |
```

Comment:
The `CLSetup` has more fields than what the table shows? Are they not meant to be changed by the user?

Reply:
Correct, unless the user really knows what they're doing ;)